### PR TITLE
http protocol should not use TLS

### PR DESCRIPTION
**Pull request by jonaz**
I need this in order to run kubectl proxy and then
make traefik use http://localhost to get to my cluster
when developing

### CODE REVIEW

**Comment by emilevauge** [comment]
Thanks @jonaz !
LGTM
/cc @vdemeester

File: cmd.go [hunk 1]
```diff
@@ -171,7 +172,7 @@ func init() {
 	traefikCmd.PersistentFlags().StringVar(&arguments.Boltdb.Prefix, "boltdb.prefix", "/traefik", "Prefix used for KV store")
 
 	traefikCmd.PersistentFlags().BoolVar(&arguments.kubernetes, "kubernetes", false, "Enable Kubernetes backend")
-	traefikCmd.PersistentFlags().StringVar(&arguments.Kubernetes.Endpoint, "kubernetes.endpoint", "127.0.0.1:8080", "Kubernetes server endpoint")
+	traefikCmd.PersistentFlags().StringVar(&arguments.Kubernetes.Endpoint, "kubernetes.endpoint", "http://127.0.0.1:8080", "Kubernetes server endpoint")
```
Comment (errm) [inline cmd.go]: I think the default should be something sane for production, could we leave this as it is.

Comment (jonaz) [inline cmd.go]: 127.0.0.1 without protocol is not sane at all. It results in error:

Get 127.0.0.1:8080/apis/extensions/v1beta1/ingresses: unsupported protocol scheme ""

Thats why i added correct scheme prefix.

Comment (errm) [inline cmd.go]: yeah your right. I forgot that this flag gets overridden here if the env vars are set... https://github.com/containous/traefik/blob/master/provider/kubernetes.go#L46..L50

So yeah this is fine, the correct thing will happen in production with very little config.

**Comment by vdemeester** [comment]
LGTM :wink:
Needs a rebase, but I'll carry, no worries :wink:

**Pull request closed without merging**